examples/workflow/qe_scf_calculation.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.    wget https://raw.githubusercontent.com/pranabdas/espresso/main/src/pseudos/Si.pz-vbc.UPF
</pre><p>Maybe we should use Pseudo Dojo URL or something from QE themselves? Or at least mention these in a comment.</p><p> </p><p>It look like we also added the file to the repo itself</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
@timurbazhirov @mrakitin once this PR is merged to the dev/stable branch, I plan to change PP file url to this repo. I think, in that way it would be more permanent than from another source (they tend to discontinue older version urls).